run_simulation.py

```diff
@@ -0,0 +1,137 @@
+"""
+run_simulation.py
+2018 Gregory Way
+
+Compare latent space arithmetic of various compression models with different
+simulated data
+
+Usage: Run on the command line with required arguments:
+
+    python run_simulation.py --data <DATA FILE>
+                             --noise <AMOUNT OF NOISE ADDED>
+                             --sample_size <NUMBER OF SAMPLES>
+                             --num_genes <NUMBER OF GENES>
+                             --out_file <SAVE LOCATION>
+
+    And several optional parameters that have reasonable defaults:
+
+        --seeds             How many times to compress input data
+        --transform         How to transform the input data file
+        --num_components    How many components to compress input down to
+        --epochs            How many times to loop through input data to train
+                            neural networks
+        --batch_size        How many samples to include after each weight
+                            update in neural network training
+        --vae_learning_rate         Variational Autoencoder learing rate
+        --adage_learning_rate       ADAGE learning rate
+        --loss              the loss function to minimize (str)
+        --verbose           Whether or not to print training progress in neural
+                            network models
+
+    Note that this script does not generate the simulated data. The data is
+    generated in scripts/util/simulate_expression.R.
+"""
+
+import argparse
+import numpy as np
+import pandas as pd
+from tybalt.data_models import DataModel
+
+parser = argparse.ArgumentParser()
+parser.add_argument('-d', '--data', help='location of data file')
+parser.add_argument('-o', '--noise', help='amount of noise injection')
+parser.add_argument('-n', '--sample_size', help='number of samples')
+parser.add_argument('-g', '--num_genes', help='number of genes')
+parser.add_argument('-f', '--out_file', help='where to save output results')
+parser.add_argument('-s', '--seeds', default=5,
+                    help='number of different seeds to run on current data')
+parser.add_argument('-t', '--transform', default='zeroone',
```

Comment:
Is this always a 'zeroone' transformation? If no what are the other options?

Reply:
No, but it should _usually_ be. I will add [choices](https://docs.python.org/2/library/argparse.html#choices) to the parser though to make these clear!